projects/packages/my-jetpack/global.d.ts

```diff
@@ -179,6 +179,14 @@ interface Window {
 				check_dns: boolean;
 			} >;
 		};
+		redBubbleAlerts: {
+			'missing-site-connection'?: null;
```

Comment:
Should `'missing-site-connection'?:` be declared as `boolean` instead of `null`?

```suggestion
			'missing-site-connection'?: boolean;
```

Reply:
Not with the way it is currently set up. I converted the red bubble notifications into the key of an array where the value is any data needed for the notification. In the case of bad install notifications, it's the display name of the plugin. The `missing-site-connection` does not need any data, so it can be left as null. Maybe there's a better default value? That's just what I have set up right now